LocalFeeder/tests/test_feeder.py

```diff
@@ -592,3 +592,35 @@ def test_inv_combined_control(federate_config):
     assert float(
         np.sum(np.abs(new_voltages.loc["87.3"] - old_voltages.loc["87.3"]))
     ) > 0.01 * float(np.abs(old_voltages.loc["87.3"]))
+
+def test_pv_setpoints(federate_config):
+    logging.info("Loading sim")
+    sim = FeederSimulator.FeederSimulator(federate_config)
+    sim.change_obj(
+        [
+            FeederSimulator.Command(
+                obj_name="PVSystem.113",
+                obj_property="irradiance",
+                val=1, 
+            ),
+            FeederSimulator.Command(
+                obj_name="PVSystem.113",
+                obj_property="Pmpp",
+                val=30, 
+            )
+
+        ]
+    )
+    sim.set_pv_output("113",20,5)
+    sim.change_obj(
+        [
+            FeederSimulator.Command(
+                obj_name="PVSystem.113",
+                obj_property="irradiance",
+                val=0.2, 
+            )
+        ]
+    )
+    sim.set_pv_output("113",20,5)
+
```

Comment:
I think we talked about this, but could we check that the pv is actually set there using a solve + getting the power injections.